src/AccessibilityInsights.SharedUx/Controls/TestTabs/ColorContrast.xaml.cs

```diff
@@ -279,14 +286,7 @@ private void HandleToggleButtonSwitch(bool isEnabled)
                 CCAMode.HandleToggleStatusChanged(isEnabled);
             }
 
-            spConfidence.Children.Clear();
```

Comment:
It's easier to just hide the confidence controls than to rebuild the children each time